scene/3d/camera.cpp

```diff
@@ -170,6 +181,27 @@ void Camera::set_perspective(float p_fovy_degrees, float p_z_near, float p_z_far
 	update_gizmo();
 	force_change = false;
 }
+void Camera::set_oblique(float p_fovy_degrees, Dictionary p_oblique_data, float p_z_near, float p_z_far) {
+	if (/* clang-format off */
+		!force_change && fov == p_fovy_degrees && p_z_near == near && p_z_far == far && mode == PROJECTION_OBLIQUE
+		&& (Vector3)p_oblique_data["normal"] == oblique_normal && (Vector3)p_oblique_data["position"] == oblique_position && (float)p_oblique_data["offset"] == oblique_offset 
+	) { /* clang-format on */
+		return;
+	}
```

Comment:
Instead of tweaking `clang-format`, I think it'll be better to break this huge condition into smaller chunks.

Reply:
yup, now that you say that... lol

Reply:
`	if (!force_change && fov == p_fovy_degrees && p_z_near == near && p_z_far == far && mode == PROJECTION_OBLIQUE) {
		return;
	}

	if (!force_change && (Vector3)p_oblique_data["normal"] == oblique_normal && (Vector3)p_oblique_data["position"] == oblique_position && (float)p_oblique_data["offset"] == oblique_offset ) {
		return;
	}`